MAINTAINERS.md

```diff
@@ -256,99 +100,9 @@ Contributions:
-If the commit descriptions are too poor to easily do this, then again, yell at
-your fellow contributors to make them better in the future.
-
-If it's *not* a trivial hotfix, you should probably make any number of release
-candidate tags (e.g. `urbit-os-vx.y.rc1`, `urbit-os-vx.y.rc2`, ..), test
-them, and after you confirm one of them is good, tag the release as
-`urbit-os-vx.y`.
-
-For Vere releases:
-
-Tag the release as `urbit-vx.y`.  The tag format should look something like
-this:
-
-```
-urbit-vx.y
-
-Note that this Vere release will by default boot fresh ships using an Urbit OS
-va.b.c pill.
-
-Release binaries:
-
-(linux64)
-https://bootstrap.urbit.org/urbit-vx.y-linux64.tgz
-
-(macOS)
-https://bootstrap.urbit.org/urbit-vx.y-darwin.tgz
-
-Release notes:
-
-  [..]
-
-Contributions:
-
-  [..]
-```
-
-Ensure the Vere release is marked as the 'latest' release and upload the two
-`.tgz` files to the release as `darwin.tgz` and `linux64.tgz`;
-this allows us to programmatically retrieve the latest release at
-[urbit.org/install/mac/latest/](https://urbit.org/install/mac/latest) and
-[urbit.org/install/linux64/latest](https://urbit.org/install/linux64/latest),
-respectively.
-
-The same schpeel re: release candidates applies here.
-
-Note that the release notes indicate which version of Urbit OS the Vere release
-will use by default when booting fresh ships.  Do not include implicit Urbit OS
-changes in Vere releases; this used to be done, historically, but shouldn't be
-any longer.  If there are Urbit OS and Vere changes to be released, make two
-separate releases.
-
-### Deploy the update
-
-(**Note**: the following steps are automated by some other Tlon-internal
-tooling.  Just ask `~nidsut-tomdun` for details.)
-
-For Urbit OS updates, this means copying the files into ~zod's %base desk.  The
-changes should be merged into /~zod/kids and then propagated through other galaxies
-and stars to the rest of the network.
-
-For consistency, I create a release tarball and then rsync the files in.
+You can get the "contributions" section by the shortlog between the last release
+and this release:
 
 ```
-$ wget https://github.com/urbit/urbit/archive/urbit-os-vx.y.tar.gz
-$ tar xzf urbit-os-vx.y.tar.gz
-$ herb zod -p hood -d "+hood/mount /=base="
-$ rsync -zr --delete urbit-urbit-os-vx.y/pkg/arvo/ zod/base
-$ herb zod -p hood -d "+hood/commit %base"
-$ herb zod -p hood -d "+hood/merge %kids our %base"
-```
-
-For Vere updates, this means simply shutting down each desired ship, installing
-the new binary, and restarting the pier with it.
-
-### Announce the update
-
-Post an announcement to urbit-dev.  The tag annotation, basically, is fine here
--- I usually add the %cz hash (for Urbit OS releases) and the release binary
-URLs (for Vere releases).  Check the urbit-dev archives for examples of these
-announcements.
-
-Post the same announcement to the group feed of Urbit Community.
+git shortlog LAST_RELEASE.. --no-merges
```

Comment:
(Very nitpicky, but) a tiny bit surprised this work, the below looks more natural to me.

```suggestion
git shortlog --no-merges LAST_RELEASE..
```